components/event-bus/internal/knative/subscription/controller/subscription/reconcile_test.go

```diff
@@ -2,29 +2,15 @@ package subscription
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"testing"
 	"time"
 
-	"knative.dev/pkg/apis"
-
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
-
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
-	evapisv1alpha1 "github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
-
-	controllertesting "github.com/knative/eventing/pkg/reconciler/testing"
-
-	duckv1beta1 "knative.dev/pkg/apis/duck/v1beta1"
-
-	messagingV1Alpha1 "github.com/knative/eventing/pkg/apis/messaging/v1alpha1"
-	eventingV1Alpha1 "github.com/knative/eventing/pkg/client/clientset/versioned/typed/eventing/v1alpha1"
-
-	messagingv1alpha1 "github.com/knative/eventing/pkg/client/clientset/versioned/typed/messaging/v1alpha1"
-	eventingv1alpha1 "github.com/kyma-project/kyma/components/event-bus/api/push/eventing.kyma-project.io/v1alpha1"
+	eventingv1alpha1 "github.com/kyma-project/kyma/components/event-bus/apis/eventing/v1alpha1"
```

Comment:
These package references both Kyma `eventing` and Kn `eventing`. A prefix would help differentiating them.

Reply:
ping @montaro